pyhoma/client.py

```diff
@@ -114,6 +114,15 @@ async def get_state(self, deviceurl: str) -> List[State]:
 
         return state
 
+    @backoff.on_exception(
+        backoff.expo, NotAuthenticatedException, max_tries=2, on_backoff=relogin
+    )
+    async def refresh_states(self) -> None:
+        """
+        Ask the box to refresh all devices states for protocols supporting that operation
+        """
+        await self.__post("/setup/devices/states/refresh")
+
```

Comment:
You don't want to check the response?

Reply:
Response is `{}` and `self.__post` checks the response for errors already.